app/src/main/java/org/alexcode/snake/Rankings.java

```diff
@@ -0,0 +1,96 @@
+package org.alexcode.snake;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import com.android.volley.Cache;
+import com.android.volley.Network;
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.toolbox.BasicNetwork;
+import com.android.volley.toolbox.DiskBasedCache;
+import com.android.volley.toolbox.HurlStack;
+import com.android.volley.toolbox.StringRequest;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Rankings extends AppCompatActivity {
+    private static final String URL_PLAYERS = "https://upgreat.ro/snake_get_rankings.php";
+    Button btnBackToMenu;
+    //list to store all players
+    List<Player> playersList;
+    //the recyclerview
+    RecyclerView recyclerView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
```

Comment:
I think it should be called onCreateRanking

Reply:
I use a predefined method here and override, cannot be renamed